tests/core/gemini/_utils/test_setup_call.py

```diff
@@ -146,10 +145,8 @@ def test_setup_call_extract(
         call_params={},
         extract=True,
     )
-    assert "tool_config" in call_kwargs and isinstance(
-        call_kwargs["tool_config"], ToolConfigDict
-    )
-    assert call_kwargs["tool_config"].function_calling_config == {
+    assert "tool_config" in call_kwargs and isinstance(call_kwargs["tool_config"], dict)
```

Comment:
I'm confused, this previous test was correct? it should always be `ToolConfigDict` during gemini extraction

Reply:
Oh, I see.  `ToolConfigDict ` is not dict.

https://github.com/google-gemini/generative-ai-python/blob/e0928fc0f239242f0cf5d69b6821188a8dac4e66/google/generativeai/types/content_types.py#L881-L882

Reply:
I reverted